BDArmory/GameModes/ModuleSpaceFriction.cs

```diff
@@ -103,12 +105,13 @@ void Start()
                         //have this called onvesselModified?
                     }
                 frictMult /= 4; //doesn't need to be 100% of thrust at max speed, Ai will already self-limit; this also has the AI throttle down, which allows for slamming the throttle full for braking/coming about, instead of being stuck with lower TwR
+                repulsors = VesselModuleRegistry.GetRepulsorModules(vessel).Count;
             }
         }
 
         public void FixedUpdate()
         {
-            if (!BDArmorySettings.SPACE_HACKS || !HighLogic.LoadedSceneIsFlight || !FlightGlobals.ready || this.vessel.packed || GameIsPaused) return;
+            if ((!BDArmorySettings.SPACE_HACKS && !AntiGravOverride) || !HighLogic.LoadedSceneIsFlight || !FlightGlobals.ready || this.vessel.packed || GameIsPaused) return;
```

Comment:
I'm not sure the logic is correct here. If `SPACE_HACKS` is disabled, but `AntiGravOverride` isn't, then you'll still be applying space friction below if `SF_FRICTION` was enabled when `SPACE_HACKS` was turned off. Is that intended?

Reply:
SF_FRICTION/SF_GRAVITY/SF_REPULSOR all get set to false when SPACE_HACKS is turned off - ln 3206 in BDArmorySetup

Reply:
OK, so is `ModuleSpaceFriction` getting added automatically even if space hacks are disabled then? It looks like it is in `MissileFire.Start`.